### DISCUSSION TITLE
Access GCLOUD_SERVICE_ACCOUNT_CREDENTIALS app level variable

### DISCUSSION THREAD

**Discussion by neeraj-repay** [codemagic-ci-cd/codemagic-docs#2779] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Kotlin Multiplatorm

### Steps to reproduce

I have two application in in code magic. (App A, App B) I am able to publish the application A to play store and unable to publish the application B to play store.
And I don't have service account json file in order to publish the app B into the play store.
Is there any way we can access the app level GCLOUD_SERVICE_ACCOUNT_CREDENTIALS from App A to App B?
Or I should recreate the json file and then add to google play console user permission

### Expected results

I should be able to access the GCLOUD_SERVICE_ACCOUNT_CREDENTIALS from App A to B

### Actual results

Unable to access app level variable

**Comment by dam-ease** [comment, marked as answer]
Hello @neeraj-repay,
Unfortunately, it's not possible to retrieve the service account from App A and use it for App B. However, there are a couple of options available to you. You can either create a new .json file or set up a separate service account. The process for both of these options is similar to the steps you followed for App A. For more detailed instructions, please refer to our helpful guide here: https://docs.codemagic.io/yaml-publishing/google-play/. We hope this information is useful to you!
